notebooks/en/rag_with_hf_and_milvus.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p>"We use the AI Act PDF, a regulatory framework for AI with different risk levels corresponding to more or less regulation, as the private knowledge in our RAG."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>